hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,45 @@
+require 'fileutils'
+
+def clear_pulp_content
+  content_dir = '/var/lib/pulp/content'
+  if File.directory?(content_dir)
```

Comment:
I'd move `stop_services` into this section so you don't stop anything unless you really need to
```suggestion
  if File.directory?(content_dir)
    stop_services

```